SimPlatform/main.cpp

```diff
@@ -3,6 +3,7 @@
 #include <QCommandLineOption>
 
 #include "mainwindow.h"
+#include "dashboard.h"
```

Comment:
Could you please separate the dashboard and the main application in two different projects? The main app will be the prototype, the dashboard should be contained in a GPA library that is used by the prototype. In the future, we will use the library to construct prototypes with a minimum of code.